src/en/charms-storage.md

```diff
@@ -415,6 +430,72 @@ provider currently supports a single pool configuration attribute:
 
 ### LXD (lxd)
 
+LXD-based models have access to the 'lxd' storage provider. The LXD provider
+does not currently have any specific configuration options.
+
+!!! Note:
+    To get an LXD version on either Ubuntu 14.04 LTS (Trusty) or Ubuntu 16.04
+    LTS (Xenial) that has the 'lxd' storage provider feature the
+    [LXD PPA][ppa-lxd] (
+    `sudo apt-add-repository -yu ppa:ubuntu-lxc/lxd-stable; sudo apt install -y lxd`
+    ) will be required.
+
+Every LXD-based model comes with a minimum of one LXD-specific Juju storage
+pool called 'lxd'. If ZFS and/or BTRFS are present when the controller is
+created then pools 'lxd-zfs' and/or 'lxd-btrfs' will also be available. The
+following output to the `juju storage-pools` command shows all three Juju
+LXD-specific pools:
+
+```no-highlight
+Name       Provider  Attrs
+loop       loop
+lxd        lxd
+lxd-btrfs  lxd       driver=btrfs lxd-pool=juju-btrfs
+lxd-zfs    lxd       driver=zfs lxd-pool=juju-zfs zfs.pool_name=juju-lxd
+rootfs     rootfs
+tmpfs      tmpfs
+```
+
+As can be inferred from the above output, for each Juju storage pool based on
+the 'lxd' storage provider there is a LXD storage pool that gets created. It
+is these LXD pools that will house the actual volumes.
+
+The LXD pool corresponding to the Juju 'lxd' pool doesn't get created until the
+latter is used for the first time (typically via the `juju deploy` command). It
+is called simply 'juju'.
+
+The command `lxc storage list` is used to list LXD storage pools. A full
+"contingent" of LXD non-custom storage pools would like like this:
+
+```no-highlight
++------------+-------------+--------+------------------------------------+---------+
+|    NAME    | DESCRIPTION | DRIVER |               SOURCE               | USED BY |
++------------+-------------+--------+------------------------------------+---------+
+| default    |             | dir    | /var/lib/lxd/storage-pools/default | 1       |
++------------+-------------+--------+------------------------------------+---------+
+| juju       |             | dir    | /var/lib/lxd/storage-pools/juju    | 0       |
++------------+-------------+--------+------------------------------------+---------+
+| juju-btrfs |             | btrfs  | /var/lib/lxd/disks/juju-btrfs.img  | 0       |
++------------+-------------+--------+------------------------------------+---------+
+| juju-zfs   |             | zfs    | /var/lib/lxd/disks/juju-zfs.img    | 0       |
++------------+-------------+--------+------------------------------------+---------+
+```
+
+To be clear, the three Juju-related pools above are for storing *volumes* that
```

Comment:
I actually thought I was belabouring this point:

> TABLE containing Juju pool names and LXD pool names (with the exception of 'juju' explained below).

> As can be inferred from the above output, for each Juju storage pool based on the 'lxd' storage provider there is a LXD storage pool that gets created. It is these LXD pools that will house the actual volumes.

> The LXD pool corresponding to the Juju 'lxd' pool doesn't get created until the latter is used for the first time (typically via the juju deploy command). It is called simply 'juju'.

Should I add a summary table (your *id est*) under the above sentence?